gather

```diff
@@ -48,11 +48,11 @@ def run(options=None):
         os.remove(result)
 
     # Opt in to include parent (second-level) domains.
-    include_parents = options.get("include-parents", False)
+    include_parents = options.get("include_parents", False)
 
     # Opt into stripping www. prefixes from hostnames, effectively
     # collapsing www.[host] and [host] into one record.
-    ignore_www = options.get("ignore-www", False)
+    ignore_www = options.get("ignore_www", False)
```

Comment:
Are hyphens for arguments no good? This will break some invocations, though it's obviously doable to update/notify downstream users if need be.

Reply:
The hyphens on the command-line still work fine, it's just that argparse will automatically transform them into underscores so that they work as Python object property names. No change on the user side should be necessary.